internal/winapi/process.go

```diff
@@ -1,10 +1,24 @@
 package winapi
 
-const PROCESS_ALL_ACCESS uint32 = 2097151
+const PROCESS_ALL_ACCESS uint32 = 0x1FFFFF
 
-// DWORD GetProcessImageFileNameW(
-//	HANDLE hProcess,
-//	LPWSTR lpImageFileName,
-//	DWORD  nSize
+const (
+	PROC_THREAD_ATTRIBUTE_JOB_LIST       uintptr = 0x2000D
+	PROC_THREAD_ATTRIBUTE_PSEUDOCONSOLE  uintptr = 0x20016
+	PROC_THREAD_ATTRIBUTE_PARENT_PROCESS uintptr = 0x20000
+)
+
+// BOOL CreateProcessAsUserW(
+// 	HANDLE                hToken,
+// 	LPCWSTR               lpApplicationName,
+// 	LPWSTR                lpCommandLine,
+// 	LPSECURITY_ATTRIBUTES lpProcessAttributes,
+// 	LPSECURITY_ATTRIBUTES lpThreadAttributes,
+// 	BOOL                  bInheritHandles,
+// 	DWORD                 dwCreationFlags,
+// 	LPVOID                lpEnvironment,
+// 	LPCWSTR               lpCurrentDirectory,
+// 	LPSTARTUPINFOW        lpStartupInfo,
+// 	LPPROCESS_INFORMATION lpProcessInformation
 // );
-//sys GetProcessImageFileName(hProcess windows.Handle, imageFileName *uint16, nSize uint32) (size uint32, err error) = kernel32.GetProcessImageFileNameW
```

Comment:
Is this function not needed anymore? I don't think it is being used anywhere currently but just making sure.

Reply:
Nope, this was originally gonna be used for job container stats but the image name is present in the SYSTEM_PROCESS_INFORMATION struct we get back from what we eventually went with to handle stats. https://github.com/microsoft/hcsshim/blob/7fa8bda4e6ba503caf0d53d0a4ee99b9a64ceed8/internal/winapi/system.go#L27